test/Microsoft.SqlTools.ServiceLayer.UnitTests/LanguageServer/SqlCompletionItemTests.cs

```diff
@@ -477,19 +181,18 @@ public void LabelShouldIncludeQuotedIdentifiersGivenTokenWithQuotedIdentifiers()
         }
 
         [Fact]
-        public void InsertTextShouldIncludeBracketGivenReservedName()
```

Comment:
>InsertTextShouldIncludeBracketGivenReservedName [](start = 20, length = 47)

still for reserve names we need bracket right? do we need a test for that scenario as well?

Reply:
Only if it's one of the types we specified - for example server/database/table/etc. 

We should only be bracket quoting things if it makes sense to do so, in these cases for identifiers that happen to share a name with a reserved word. 

I updated the bracket quoted test to check the reserved names too though since that should be checking that.